app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -17,17 +17,6 @@ import org.threeten.bp.Instant
 import org.threeten.bp.LocalDate
 import java.util.UUID
 
-/**
```

Comment:
This block was removed and inlined in a Migration, which was the only place where we needed to retain it.